jconstraints-benchmarktest/src/main/java/tools/aqua/jconstraints/benchmarktest/benchmarks/TestCase.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 United States Government, as represented by the Administrator
+ *                of the National Aeronautics and Space Administration. All Rights Reserved.
+ *           2017-2021 The jConstraints Authors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tools.aqua.jconstraints.benchmarktest.benchmarks;
+
+import gov.nasa.jpf.constraints.api.ConstraintSolver;
+import gov.nasa.jpf.constraints.api.ConstraintSolver.Result;
+import gov.nasa.jpf.constraints.api.Expression;
+
+/**
+ * Contains a {@link Expression<Boolean>} which can be tested by a {@link ConstraintSolver}
+ * and an expected {@link Result}. 
+ */
+public interface TestCase {
+
+	
+/**
+ * Returns the {@link Expression} of the current {@link TestCase}.
+ * 
+ *  @return the {@code Expression<Boolean>} of the current {@code TestCase} 
+ */
+Expression<Boolean> getTest();
+
+  /**
+   * Returns the expected {@link Result} of the current {@link TestCase}.
+   * 
```

Comment:
Whitespace broken

Reply:
Weird as i used the Google Format Task.

Reply:
For me, the task does reformat 7 files. Setup issue?

Reply:
Could be. I'm looking into it.

Reply:
Ok I fixed it, now it changed 7 files for me as well.